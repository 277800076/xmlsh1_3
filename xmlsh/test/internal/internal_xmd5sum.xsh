# Test of xmd5sum

echo "Hello World" | xmd5sum
xmd5sum ../../samples/data/*.xml
