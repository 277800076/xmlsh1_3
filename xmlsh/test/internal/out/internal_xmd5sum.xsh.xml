<xmd5>
  <file md5="3579c8da7f1e0ad94656e76c886e5125" length="13"/>
</xmd5>
