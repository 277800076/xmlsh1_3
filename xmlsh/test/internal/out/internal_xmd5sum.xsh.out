<xmd5>
  <file md5="3579c8da7f1e0ad94656e76c886e5125" length="13"/>
</xmd5>
<xmd5>
  <file name="../../samples/data/REC-xml-20060816.xml" md5="8fac09521dfbbf3509024a924351f49f" length="199019"/>
  <file name="../../samples/data/books.xml" md5="dbd4c7ab88e5092f7ca83643cef2fd86" length="2841"/>
  <file name="../../samples/data/intl.xml" md5="73696927038bcac47b0c87d3aec16bc2" length="418"/>
  <file name="../../samples/data/othello.xml" md5="7dd2133a6e6104d79e017cd7062aa939" length="248777"/>
  <file name="../../samples/data/xi_doc1.xml" md5="80e03792e5d21285672ad5295f54c989" length="297"/>
  <file name="../../samples/data/xi_sub1.xml" md5="57024d1dc619fc7db898f10083d62a93" length="212"/>
  <file name="../../samples/data/xml-base-chap.xml" md5="3b2cf4f7c2b22090de0def548967b97e" length="101"/>
  <file name="../../samples/data/xml-base-test.xml" md5="09d39f674b7d2ce9c7b967d408d185bf" length="203"/>
</xmd5>
